packages/siopv2-openid4vp-example-backend/src/presentationDefinitions.ts

```diff
@@ -3,98 +3,128 @@ import {Rules} from "@sphereon/pex-models";
+      name: "Wallet Identity",
+      rule: Rules.Pick,
+      min: 2,
+      max: 3,
+      from: "A"
+    }],
+    input_descriptors: [/*{
+        id: "walletId",
+        purpose: "Checking your Wallet information",
+        name: "Wallet Identity",
+        group: ["A"],
+        schema: [{uri: "https://sphereon-opensource.github.io/ssi-mobile-wallet/context/sphereon-wallet-identity-v1.jsonld"}]
             },*/
+      {
+        id: "WorkplaceCredential",
+        name: "ProfessionalCredential",
+        purpose: "We need to verify your employment for the demo",
+        group: ["A"],
+        schema: [
+          {
+            uri: "WorkplaceCredential" // ENTRA
+          },
+          {uri: "https://sphereon-opensource.github.io/ssi-mobile-wallet/context/sphereon-wallet-identity-v1.jsonld"} // Sphereon
+
+        ],
+        constraints: {
+          fields: [
             {
-                "id": "WorkplaceCredential",
-                "name": "ProfessionalCredential",
-                "purpose": "We need to verify your employment for the demo",
-                group: ["A"],
-                "schema": [
-                    {
-                        "uri": "WorkplaceCredential" // ENTRA
-                    },
-                    {uri: "https://sphereon-opensource.github.io/ssi-mobile-wallet/context/sphereon-wallet-identity-v1.jsonld"} // Sphereon
+              path: [
+                "$.issuer",
+                "$.vc.issuer",
+                "$.iss"
+              ],
+              filter: {
+                "type": "string",
+                "pattern": "did.*"
+              }
+            }
+          ]
+        }
+      },
+      {
+        id: "OpenBadgeCredential",
+        name: "AchievementCredential",
+        purpose: "We need to verify your achievement for the demo",
+        group: ["A"],
+        schema: [
+          {
+            uri: "AchievementCredential" // ENTRA
+          },
+          {uri: "https://purl.imsglobal.org/spec/ob/v3p0/context.json"}
 
-                ],
-                "constraints": {
-                    "fields": [
-                        {
-                            "path": [
-                                "$.issuer",
-                                "$.vc.issuer",
-                                "$.iss"
-                            ],
-                            "filter": {
-                                "type": "string",
-                                "pattern": "did.*"
-                            }
-                        }
-                    ]
-                }
-            }]
-    };
+        ],
+        constraints: {
+          fields: [
+            {
+              path: [
+                "$.issuer",
+                "$.vc.issuer",
+                "$.iss",
+                "$.issuer.id",
+              ],
+              filter: {
+                "type": "string",
+                "pattern": "did:key:z6Mko7jcpj8eeUXPbrtKGrSsYzjjuV84HnzuhSkbngpFnMrV"
```

Comment:
How would this filter match all the issuers available in JFF?

Reply:
@nklomp 
The ticket only mentions one issuer. 
... using Open Badge Credential **(from WaltID)** 

I take it from this comment that we want all JFF Issuers. What does it mean ? Does it mean that we allow all openBadge credentials ? Should following be sufficient to classify all JFF issuers ?

"pattern": "did.*"

Reply:
Yes multiple issues. Either including that pattern, or leaving out that filter altogether.

Reply:
And please think a bit about the goals of a ticket instead of blindly interpreting things. Discuss with me if things are unclear.